cernopendata/static/templates/search/briefs/software.html

```diff
@@ -6,16 +6,16 @@ <h4>
     </h4>
     <p ng-if="record.metadata.abstract " ng-bind-html="record.metadata.abstract | limitTo : 200 | unsafe" class="mt-1"></p>
     <div class="tags-box-search-results">
-      <a class="badge badge-success custom-badge-success" href="/search?type={{record.metadata.type}}">
+      <a class="badge badge-success custom-badge-success" ng-if="record.metadata.type" href="/search?type={{record.metadata.type}}">
         {{ record.metadata.type }}
       </a>
-      <a class="badge badge-info" ng-if="record.subtype" href="/search?subtype={{record.metadata.subtype}}">
+      <a class="badge badge-info" ng-if="record.metadata.subtype" href="/search?subtype={{record.metadata.subtype}}">
```

Comment:
@ioannistsanaktsidis While you are at it, can you please also change the following occurrence of `record.subtype`:

```console
$ git grep -A 2 -Hni record.subtype
cernopendata/static/templates/search/briefs/dataset.html:12:      <a class="badge badge-info" ng-if="record.subtype" href="/search?subtype={{record.metadata.subtype}}">
cernopendata/static/templates/search/briefs/dataset.html-13-        {{ record.metadata.subtype }}
cernopendata/static/templates/search/briefs/dataset.html-14-      </a>
```

The `dataset` briefrs are not really used right now, just as the `software` briefs, but if you modify them in a bulk mode, let's better be ready when we may plug it in later.

P.S. I see some more cases we may want to decorate with `ng-if`:

```
<a class="badge badge-success custom-badge-success" href="/search?type={{record.metadata.type}}">
cernopendata/static/templates/search/briefs/dataset.html:      <a class="badge badge-warning custom-badge-warning" href="/search?experiment={{record.metadata.experiment}}">
```

This can come later when we shall introduce strict distinction between required and optional JSON schema fields.  (CC @ArtemisLav)

Otherwise LGTM 👍